config/locales/user_mailer/es.yml

```
@@ -175,19 +173,19 @@ es:
       call_to_action: 'Como medida de precaución, inhabilitamos su contraseña para
         mantener su información segura. Siga los pasos a continuación para
         restablecer su contraseña y asegure su cuenta:'
-      intro: Hemos detectado actividad inusual en su cuenta login.gov. Estamos
+      intro: Hemos detectado actividad inusual en su cuenta Login.gov. Estamos
         preocupados alguien que no sea usted puede estar intentando acceder a su
         información.
       learn_more_link_text: Conozca más sobre sus opciones
-      reminder_html: Le recordamos que <b>login.gov nunca le pedirá sus credenciales
+      reminder_html: Le recordamos que <b>Login.gov nunca le pedirá sus credenciales
         de inicio de sesión. por teléfono o correo electrónico</b>. Puede tomar
         medidas adicionales para proteger su cuenta habilitando la autenticación
         de dos factores.
-      step_1: Visite el sitio web login.gov y seleccione iniciar sesión
+      step_1: Visite el sitio web Login.gov y seleccione iniciar sesión
       step_2: Seleccione “¿olvidó su contraseña?” al final de la página
       step_3: Siga las instrucciones para restablecer su contraseña
-      subject: Actividad inusual — restablezca su contraseña de login.gov
-    reset_password_instructions:
+      subject: Actividad inusual — restablezca su contraseña de Login.gov
+      reset_password_instructions:
```

This should be a level up:

```suggestion
    reset_password_instructions:
```